src/apps/airswap/airswap.module.ts

```diff
@@ -0,0 +1,18 @@
+import { Module } from '@nestjs/common';
+
+import { AbstractDynamicApp } from '~app/app.dynamic-module';
+
+import { AirswapAppDefinition } from './airswap.definition';
+import { AirswapContractFactory } from './contracts';
+import { EthereumAirswapBalanceFetcher } from './ethereum/airswap.balance-fetcher';
```

Comment:
Is airswap.balance-fetcher necessary? Since airswap tokens are already shown on dashboard.